app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -0,0 +1,293 @@
+            <div class="title-block text-base font-medium">
+              <span class="mr-1">{{ websiteName }}</span>
+              <div v-if="isOnline" class="online-dot"></div>
+            </div>
+            <div class="text-xs mt-1 text-black-700">
+              {{ responseTime }}
+            </div>
+          </div>
+        </div>
+        <div v-if="isExpanded" class="header-expanded">
+          <h2 class="text-slate-900 mt-6 text-4xl mb-3 font-normal">
+            {{ welcomeHeading }}
+          </h2>
+          <p class="text-lg text-black-700 leading-normal">
+            {{ welcomeTagLine }}
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="conversation--container">
+      <div class="conversation-wrap">
+        <div class="message-wrap">
+          <div class="user-message-wrap">
+            <div class="user-message">
+              <div class="message-wrap">
+                <div class="chat-bubble user">
+                  <p>Hello</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="agent-message-wrap">
+          <div class="agent-message">
+            <div class="avatar-wrap"></div>
+            <div class="message-wrap">
+              <div class="chat-bubble agent">
+                <div class="message-content">
+                  <p>Hello</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="footer-wrap">
+      <div class="input-wrap">
+        <footer class="footer">
+          <div class="chat-message--input">
+            <text-area
+              aria-placeholder="Type your message"
+              class="form-input user-message-input"
+            />
+            <div class="button-wrap">
+              <span class="file-uploads file-uploads-html5">
+                <span class="attachment-button">
+                  <i class="ion-android-attach"></i>
+                </span>
+              </span>
+            </div>
+          </div>
+        </footer>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Widget',
+  props: {
+    welcomeHeading: {
+      type: String,
+      default: 'Hi There,',
+    },
+    welcomeTagLine: {
+      type: String,
+      default: '',
+    },
+    websiteName: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    websiteDomain: {
+      type: String,
+      default: '',
+    },
+    logo: {
+      type: String,
+      default: '',
+    },
+    isExpanded: {
+      type: Boolean,
+      default: true,
+    },
+    isOnline: {
+      type: Boolean,
+      default: true,
+    },
+    replyTime: {
+      type: String,
+      default: 'few minutes',
+    },
+  },
+  computed: {
+    // eslint-disable-next-line func-names
+    responseTime: function() {
+      return `Typically replies in a ${this.replyTime}`;
```

Comment:
We could use expect the replyTime to be something like this in `app/javascript/widget/mixins/availability.js`. As we need to make use of translations.
```
        case 'in_a_few_minutes':
          return this.$t('REPLY_TIME.IN_A_FEW_MINUTES');
        case 'in_a_few_hours':
          return this.$t('REPLY_TIME.IN_A_FEW_HOURS');
        case 'in_a_day':
          return this.$t('REPLY_TIME.IN_A_DAY');
        default:
          return this.$t('REPLY_TIME.IN_A_FEW_HOURS');

```